packages/serverpod/lib/src/server/tasks/task.dart

```diff
@@ -0,0 +1,23 @@
+/// A task that can be executed by the [TaskManager].
+///
+/// Tasks are managed by the [TaskManager] and can be executed at various points,
+/// such as during server shutdown. Tasks can be executed in parallel or
+/// sequentially based on their configuration.
```

Comment:
```suggestion
/// such as during server shutdown.
```

I think, as there is nothing in the `Task` that would configure this and for now we only have 1 task manager that also does not handle any configuration.